installer-config.txt

```diff
@@ -8,7 +8,7 @@ userpw=raspberry
 boot_volume_label=openHABian
 disable_root=1
 user_is_admin=1
-username=pi
+username=admin
```

Comment:
It doesn't matter for me.  Changed from `pi` because a) it is standard for RPi and more prone for brute-forces but as BClark09 mention `admin` is not that good either. b) would be in my opinion illogical to have username `pi` for managing openhab.

But in the end of the day it's just preferences. The ones (like me) who wants a different name can just change it.